index.html

```diff
@@ -34,5 +34,112 @@ <h2 class="subtitle">I’am a Software Developer</h2>
+        <button class="project-button" type="button">See Project</button>
+        </section>
+        <section class="container">
+            <h2 class="project-tittle" >Profesional Art Printing Data</h2>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="project-tools">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+        <button class="project-button" type="button">See Project</button>
+        </section>
+        <section class="container">
+            <h2 class="project-tittle" >Profesional Art Printing Data</h2>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="project-tools">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+        <button class="project-button" type="button">See Project</button>
+        </section>
+        <section class="container">
+            <h2 class="project-tittle" >Profesional Art Printing Data</h2>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="project-tools">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+        <button class="project-button" type="button">See Project</button>
+        </section>
+    </section>
+        <section class="aboutMe">
+            <h2 class="aboutMe-tittle">About me</h2>
+            <p class="aboutMe-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button class="aboutMe-button" type="button">Get My Resume</button>
+            <hr class="abouteMe-line">
+        </section>
+        <section class="languages">
+            <h3>Language</h3>
+            <ul class="tools-lists">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+            <h3>Frameworks</h3>
+            <ul class="tools-lists">
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+            </ul>
+            <h3>Skills</h3>
+            <ul class="tools-lists">
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </section>
```

Comment:
Hey 👋🏽 You did everything well but you just left out one thing in the design 😓 The shapes in front of the titles, can you add them? 👇🏽 

## Figma
![image](https://user-images.githubusercontent.com/45739963/219906207-58e8b209-051c-490e-88d0-872d191e06a3.png)

## Your design
![image](https://user-images.githubusercontent.com/45739963/219906230-6feab650-3131-4313-a36e-22da83afcf6a.png)